reframe/views/All.css

```diff
@@ -56,7 +56,6 @@ a:hover
 
 .container-lg {
   max-width: 1012px;
-  width: 1012px;
```

Comment:
This change, together with removing `<div className="container-lg">` in `NavBar.js` breaks the navbar on desktop: now the navbar buttons end up both on the left of the screen.